ppl/zqd/core.go

```diff
@@ -136,6 +141,13 @@ func (c *Core) addWorkerRoutes() {
 	c.handle("/worker", handleWorker).Methods("POST")
 }
 
+func (c *Core) addRecruiterRoutes() {
```

Comment:
Nits: Keep methods sorted by putting this between addAPIServerRoutes and addWorkerRoutes.